README.md

```diff
@@ -4,21 +4,48 @@ A modern web-based Pokemon save file editor and parser built with React, TypeScr
 
```

Comment:
lets make the description less braggy. just state what it is, a web based savegame editor, cli and typescript parser core